SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverDebugContext.cpp

```diff
@@ -142,24 +205,44 @@ void USpatialNetDriverDebugContext::RemoveActorTag(AActor* Actor, FName Tag)
 	}
 }
 
-void USpatialNetDriverDebugContext::OnDebugComponentUpdateReceived(Worker_EntityId Entity)
+void USpatialNetDriverDebugContext::AddComponent(Worker_EntityId EntityId)
 {
-	SpatialGDK::DebugComponent* DbgComp = NetDriver->StaticComponentView->GetComponentData<SpatialGDK::DebugComponent>(Entity);
+	const SpatialGDK::EntityViewElement& Element = NetDriver->Connection->GetView().FindChecked(EntityId);
```

Comment:
Why are we using the connection view and not the subview here?

Reply:
For the actual answer, when addressing above comments I was surprised that we would grab a view through the netdriver that wasn't through the connection. So rather than clocking that there was a subview owned by the netdriver and fixing that I changed it to connection. Then I went back and fixed the subview thing and left this. Change to the subview